clang/include/clang/Driver/Options.td

```diff
@@ -4082,7 +4086,9 @@ def ffinite_loops: Flag<["-"],  "ffinite-loops">, Group<f_Group>,
 def fno_finite_loops: Flag<["-"], "fno-finite-loops">, Group<f_Group>,
   HelpText<"Do not assume that any loop is finite.">,
   Visibility<[ClangOption, CC1Option]>;
-
+def fthinlto_distributor_EQ : Joined<["-"], "fthinlto-distributor=">, Group<f_Group>,
+  HelpText<"Specifies the <path> to the distributor process executable.">, MetaVarName<"<path>">,
```

Comment:
No trailing `.` (the majority convention of HelpText )

Reply:
Thanks. Done.